ou_dedetai/tui_app.py

```diff
@@ -476,17 +476,16 @@ def choice_processor(self, stdscr, screen_id, choice):
             2: self.handle_ask_response,
             8: self.waiting,
             10: self.waiting_releases,
-            11: self.winetricks_menu_select,
-            12: self.logos.start,
-            13: self.waiting_finish,
-            14: self.waiting_resize,
-            15: self.password_prompt,
-            18: self.utilities_menu_select,
-            19: self.renderer_select,
-            20: self.win_ver_logos_select,
-            21: self.win_ver_index_select,
-            24: self.confirm_restore_dir,
-            25: self.choose_restore_dir,
+            11: self.logos.start,
+            12: self.waiting_finish,
+            13: self.waiting_resize,
+            14: self.password_prompt,
+            15: self.utilities_menu_select,
+            16: self.renderer_select,
```

Comment:
These numbers are used elsewhere in the code as indexes, ideally these should be moved to constants, however in the meantime making sure the numbers don't change (even if there is gaps) is good enough for now.

Reply:
Oh, I'll put those back, then. I didn't realize the indexes weren't simply sequential.